_pages/contributing.md

```diff
@@ -359,6 +357,93 @@ dismissive remarks will not be tolerated.**
 
 When you understand what changes are required make amendments to the pull
 request by adding patches to the same branch and then pushing to the forked repository.
+A git "force push" (i.e., `git push --force`) is also acceptable if you wish to alter the commits locally in order to make the changes.
 
 Once pushed to the forked repository, the pull request will automatically update with your changes.
-A reviewer will then review your changes and, if necessary, ask for further changes, or approve your pull-request.
+The reviewer will then re-review your changes and, if necessary, ask for further changes, or approve your pull-request.
+
+## Reviewing other contributions
+
+We encourage all gem5 developers to review other's contributions.
+Anyone may review a gem5 change and, if they feel it is ready, approve it.
+All pull-requests can be found at <https://github.com/gem5/gem5/pulls>.
+
+When reviewing a pull request we enforce the followings guidelines.
+These have been designed to ensure clear and polite communication between all parties:
+
```

Comment:
Putting this link would be good, too: https://google.github.io/eng-practices/review/reviewer/comments.html